checkClock/config/routes.rb

```diff
@@ -1,3 +1,4 @@
 Rails.application.routes.draw do
+  get 'home/index'
```

Comment:
Are you going to use that route for home? I think you can explain a more concrete route for your dashboard.

```suggestion
Rails.application.routes.draw do
  get 'home/index', as: 'dashboard'
```